### DISCUSSION TITLE
How to delete waypoints?

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#157] [General]
Hi Kjell, 

I am setting some waypoints for the units movement, but I made a mistake and I would delete some of them. I tried in some ways, but I am now unable to delete (or select for deletion) a waypoint after creating it. Can you suggest me a way to get ridden of waypoints in excess?

Thanks
Federico

**Comment by kjellmf** [comment]
I'm sorry for the unintuitive user interface. I have not done any work on this part or ORBAT mapper for a while now.  Try using the Alt key while clicking on a vertex/waypoint. You can also use the "dots" menu in the state section of the unit panel:


![image](https://github.com/orbat-mapper/orbat-mapper/assets/29217/d2f9ada2-726c-4cea-b40f-5a094be55b03)

**Reply by kjellmf** [reply]
Alt+click does not seem to work properly. I'll add an issue.

**Reply by federicopeyrani1969** [reply]
I observed that wih alt+clik key you can't delete the end of the unit path, but you can delete points in the middle of it, so it's helpful anyway.
Please note that not all the points of a path appear in the list but, ist seems to me, only the ones that I set with the "Set Location" command.
Thank you for your help
Federico

**Comment by kjellmf** [comment]
I've fixed a few bugs in the waypoint editor. It should now be possible to delete waypoints with Ctrl+click and by selecting them and using the `del`button
